covfee/client/tasks/continuous_annotation/spec.ts

```diff
@@ -20,6 +20,7 @@ export interface ContinuousAnnotationTaskSpec extends BaseTaskSpec {
   media: MediaSpec[]
   annotations: AnnotationDataSpec[]
   prolificCompletionCode?: string
+  taskIntroInstructions?: string[]
```

Comment:
I did not quite understand why a list was required. Perhaps renaming to `shortInstructionsPopupBulletPoints` could help ?

Reply:
Fixed in 0772750d37dc1dcc5c873945250f37d1afe68936